trento/xml/article_sap_trento.xml

```diff
@@ -256,29 +271,36 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
       Trento is powered by event driven technology. Registered events are stored in a  &postgresql; database with a retention
       period of 10 days. For each host registered with Trento, you need to allocate at least 1.5GB of space in the &postgresql; database.
      </para>
-      <para> While the &t.server; supports various usage scenarios,
-        depending on the existing infrastructure, it is designed to be
-        cloud native and OS agnostic. It can
-        be installed on the following services: </para>
+          <para>
+      Prometheus is used by Trento Server to collect metric data collected by the exporters in the agent hosts.
+           It is an optional component and only required if we want to have graphical charts of the collected metrics 
+           in the console. Otherwise we can disable the charts upon installation. We can perform an ad-hoc installation of
+           Prometheus for Trento or use an existing one. In the case of a systemd deployment on SLES for SAP SP3, only the latter is possible.
```

Comment:
Not sure if this is useful, but I would slightly rephrase it:

```suggestion
           This is an optional component that is only required if the collected metrics are to be displayed graphically
           in the console. Otherwise, we can disable the charts during installation. We can perform an ad-hoc installation of
           Prometheus for Trento or use an existing one. In the case of a systemd deployment on &sles4sap; SP3, only the latter is possible.
```

Reply:
I cannot commit the suggestion because it says it is outdated but the rephrasing is OK with me.

Reply:
I'm afraid I do not know how to work locally with git. I do everything directly on the web site.

Reply:
It should work on the web too. Sometimes the GitHub page needs a refresh.